.drone.yml

```diff
@@ -56,7 +56,7 @@ pipeline:
       event: [ push, tag, pull_request ]
 
   build-without-gcc:
-    image: golang:1.9
```

Comment:
Can you keep this pegged at 1.9 as it is currently the oldest version of go that is supported.

Reply:
(but the others can remain at 1.11 as we just need to ensure that 1.9 builds, but we will distribute the versions of gitea that are built with a more recent version)